### ISSUE TITLE
Join Animations Messes up Previous Animations

### ISSUE THREAD

**Issue by stryker313** [vini-guerrero/Godot_Game_Tools#40]
Basically, upon joining another animation, the previous animations are moved up off the ground, and their feet are not longer planted and the feet float. 

Video: https://youtu.be/_OCqOw-_Lss

**Comment by caniko** [comment]
Yes, this happens to me too. I have to import all animations in join animation for mitigation.

**Comment by stryker313** [comment]
I found that using Rokoko Retargeter instead of join animations is better than using the default join. Simply retarget all the anims that you want, then export using godot games tool export. Works like a charm.

**Comment by caniko** [comment]
Very smart, I'll try that out!
